shared/src/commonMain/kotlin/me/saket/press/shared/rx/TakeWhile.kt

```diff
@@ -1,36 +0,0 @@
-package me.saket.press.shared.rx
-
-import com.badoo.reaktive.base.tryCatch
-import com.badoo.reaktive.completable.CompletableCallbacks
-import com.badoo.reaktive.disposable.Disposable
-import com.badoo.reaktive.disposable.DisposableWrapper
-import com.badoo.reaktive.observable.Observable
-import com.badoo.reaktive.observable.ObservableObserver
-import com.badoo.reaktive.observable.observable
-
-/**
- * Emit items by upstream as long as `predicate` is satisfied,
- * and then completes as soon as this condition is not satisfied.
- */
-fun <T> Observable<T>.takeWhile(predicate: (T) -> Boolean): Observable<T> =
```

Comment:
Is there a issue request for Kotlin native compiler?

Reply:
No, I should create one. Thank you for the reminder.

Reply:
Done: https://youtrack.jetbrains.com/issue/KT-35477